src/main/resources/db/reference/security.sql

```diff
@@ -1,405 +1,429 @@
-insert into security_role_permission(rop_id,rol_id,per_id) values (40,5,6);
-insert into security_role_permission(rop_id,rol_id,per_id) values (41,5,7);
-insert into security_role_permission(rop_id,rol_id,per_id) values (42,5,10);
-insert into security_role_permission(rop_id,rol_id,per_id) values (43,5,13);
-insert into security_role_permission(rop_id,rol_id,per_id) values (44,5,14);
-
-insert into court_case(cas_id,cth_id,case_number,interpreter_used) values(1,1,'U01012023-00001',false);
-insert into court_case(cas_id,cth_id,case_number,interpreter_used) values(2,1,'U01012023-00002',true);
-insert into court_case(cas_id,cth_id,case_number,interpreter_used) values(3,1,'U01012023-00003',false);
-insert into court_case(cas_id,cth_id,case_number,interpreter_used) values(4,2,'U01012023-00004',true);
-insert into court_case(cas_id,cth_id,case_number,interpreter_used) values(5,2,'U01012023-00005',false);
-insert into court_case(cas_id,cth_id,case_number,interpreter_used) values(6,3,'U01012023-00006',true);
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+--v12 remove synthetic primary key from associative entity security_group_membership
+--    remove columns users_names and r_dm_group_s_object_id from security_group_membership
+--    change name of security_group_membership to security_group_ae
+--    remove synthetic primary key from security_role_permission and add _ae suffix 
+--    remove synthetci primary key from security_group_courthouse and add _ae suffix
+--    explicitly create indexes for primary keys with storage clauses, then utilise those indexes to define PKs
+--    add sequences for remaining synthetic primary keys
+--    amend type of r_modify_date to timestamp with time ZONE
+--    amend is_private to BOOLEAN
+
+
+-- List of Table Aliases
+
+--security_group                       GRP
+--security_group_membership_ae         GRM
+--security_role                        ROL
+--security_permission                  PER
+--security_role_permission_ae          ROP
+--security_group_courthouse_ae         GRC
+
+SET ROLE DARTS_OWNER;
+SET SEARCH_PATH TO darts;
+
+CREATE TABLE security_group
+(grp_id                  INTEGER
+,rol_id                  INTEGER
+,r_dm_group_s_object_id  CHARACTER VARYING(16)
+,group_name              CHARACTER VARYING
+,is_private              BOOLEAN
+,description             CHARACTER VARYING
+,r_modify_date           TIMESTAMP WITH TIME ZONE
+,group_class             CHARACTER VARYING
+,group_global_unique_id  CHARACTER VARYING
+,group_display_name      CHARACTER VARYING
+);
+
+COMMENT ON TABLE security_group 
+IS 'migration columns all sourced directly from dm_group_s, additional attributes may be required from dm_user_s, but data only where dm_user_s.r_is_group=1';
+COMMENT ON COLUMN security_group.grp_id
+IS 'primary key of security_group';
+COMMENT ON COLUMN security_group.r_dm_group_s_object_id
+IS 'internal Documentum primary key from dm_group_s';
+
+CREATE TABLE security_group_membership_ae

```

Comment:
Wondering would convention for this table name be `user_account_security_group_ae`?